triton/checkpoint.py

```diff
@@ -4,26 +4,54 @@
 import psycopg2.pool
 import time
 
+from triton import errors
+
 log = logging.getLogger(__name__)
 
 
 triton_dsn = os.environ.get('TRITON_DB')
+triton_client_name = os.environ.get('TRITON_CLIENT_NAME')
 postal_rds_pool = None
 
+CREATE_TABLE_STMT = """
+CREATE TABLE IF NOT EXISTS triton_checkpoint (
+    client VARCHAR(255) NOT NULL,
+    stream VARCHAR(255) NOT NULL,
+    shard VARCHAR(255) NOT NULL,
+    seq_num VARCHAR(255) NOT NULL,
+    updated INTEGER NOT NULL,
```

Comment:
Same comment as before about the not null constraints-- and yes, I did mean the current checkpoint db.